config/codon_dbs.config

```diff
@@ -14,4 +14,6 @@ params {
     kegg_classes = "/hps/nobackup/rdf/metagenomics/service-team/production/ref-dbs/genomes-pipeline/kegg_classes.tsv"
 
     amrfinder_plus_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/amrfinderplus/3.11/2023-02-23.1"
+    
+    checkm2_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/checkm2/CheckM2_database/CheckM2_database/uniref100.KO.1.dmnd"
```

Comment:
We need to normalize that path, it should be
```
/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/checkm2/<version>/uniref100.KO.1.dmnd
```

Reply:
Good catch, I updated the path.